components/console/helpers/questionhelper.rst

```diff
@@ -216,6 +216,30 @@ provide a callback function to dynamically generate suggestions::
 
     The ``setAutocompleterCallback()`` method was introduced in Symfony 4.3.
 
+Allow to not trim the response
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also specify if you want to not trim the answer by setting it directly with
+:method:`Symfony\\Component\\Console\\Question\\Question::setTrimmable`::
+
+    use Symfony\Component\Console\Question\Question;
+
+    // ...
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        // ...
+        $helper = $this->getHelper('question');
+
+        $question = new Question('What is the name of the child?');
+        $question->setTrimmable(true);
+        // if the users inputs 'elsa ' it will not be trimmed and you will get 'elsa ' as value
```

Comment:
the comment is wrong regarding the line above. If i set trimmable=true, it will be trimmed right?